libraries/core_libs/network/include/network/tarcap/packets_handlers/pbft_block_packet_handler.hpp

```diff
@@ -6,10 +6,13 @@ namespace taraxa {
 class PbftBlock;
 class PbftChain;
 class PbftManager;
+class DagBlockManager;
 }  // namespace taraxa
 
 namespace taraxa::network::tarcap {
 
+class SyncingState;
```

Comment:
why are these forwards declarations in this header when they are not used here ?